src/mds/MDSRank.cc

```diff
@@ -2734,7 +2734,12 @@ void MDSRankDispatcher::handle_asok_command(
     r = config_client(client_id, !got_value, option, value, *css);
   } else if (command == "scrub start" ||
 	     command == "scrub_start") {
-    if (whoami != 0) {
+    if (!is_active()) {
+      *css << "MDS is not active";
+      r = -CEPHFS_EINVAL;
+      goto out;
+    }
+    else if (whoami != 0) {
```

Comment:
do `scrub pause|resume|abort` commands need to handle this case as well ?

Reply:
I think yes, they also need to be handled. Thanks for pointing. @vshankar ?

Reply:
Probably. Try to pause a scrub (which with this change it cannot be run) in s-r and see what happens. My guess it it might throw an error and not crash.